apps/src/sites/studio/pages/levels/editors/_applab.js

```diff
@@ -62,7 +63,19 @@ $(document).ready(function() {
     }
   });
 
+  const styles = {
+    optgroup: {
+      color: color.lightest_gray
+    },
+    option: {
+      color: color.black
+    }
+  };
+
   const data = getScriptData('applabOptions');
+  const categories = (data.dataset_library_manifest.categories || []).filter(
+    category => category.published
```

Comment:
thanks @ajpal ! just checking that (1) showing only published categories is correct, and (2) is there something that keeps the set of datasets within published categories in sync with the list of `dataset_library_manifest.tables`?

Reply:
Yes, I think showing only published tables here is correct.
There is not currently anything that keeps published tables in sync with the library manifest. The idea with having the `published` flag on the tables/categories was so that levelbuilders could test out new tables without them going live immediately, but it's worth reconsidering if that's still a used/desired function.